### DISCUSSION TITLE
có gì khó khăn không mnggg

### DISCUSSION THREAD

**Discussion by NKLinhNgan-10A02-stt19** [phhnhan-10A02-stt21/lop-10A02-Nhom8#20] [General]
có dì hõi hõi đi mngggg

**Comment by maidangiu** [comment]
có khó khăn sương sương

**Comment by maidangiu** [comment]
chạy dl mệt qá mốt ling ngưn nhớ bao chà xữa

**Reply by 10A2DucAn01** [reply]
+999

**Reply by NKLinhNgan-10A02-stt19** [reply]
làm tin đi r bao chà xữa

**Comment by 10A2DucAn01** [comment]
có cái xin câu hỏi thoi mà tỉ gửi qua phần issue ròi đó tỉ

**Reply by NKLinhNgan-10A02-stt19** [reply]
rúp học giỏi quá k có dì khó luôn::))
